superset/assets/javascripts/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -0,0 +1,558 @@
+                "annotation. If you don't select a column all of them will be shown. "}
+                multi
+                options={
+                  columns
+                }
+                value={descriptionColumns}
+                onChange={v => this.setState({ descriptionColumns: v })}
+              />
+            }
+            <div style={{ marginTop: '1rem' }}>
+              <CheckboxControl
+                hovered
+                name="annotation-override-since"
+                label="Pass down 'Since'"
+                description={'This controls whether the "Since" field from the current ' +
+                  'view should be passed down to the slice containing the annotation data.'}
+                value={!!Object.keys(overrides).find(x => x === 'since')}
+                onChange={(v) => {
+                  delete overrides.since;
+                  if (v) {
+                    this.setState({ overrides: { ...overrides, since: null } });
+                  } else {
+                    this.setState({ overrides: { ...overrides } });
+                  }
+                }}
+              />
+              <CheckboxControl
+                hovered
+                name="annotation-override-until"
+                label="Pass down 'Until'"
+                description={'This controls whether the "Until" field from the current ' +
+                'view should be passed down to the slice containing the annotation data.'}
+                value={!!Object.keys(overrides).find(x => x === 'until')}
+                onChange={(v) => {
+                  delete overrides.until;
+                  if (v) {
+                    this.setState({ overrides: { ...overrides, until: null } });
+                  } else {
+                    this.setState({ overrides: { ...overrides } });
+                  }
+                }}
+              />
+            </div>
+          </PopoverSection>
+        </div>
+      );
+    }
+    return ('');
+  }
+
+  renderDisplayConfiguration() {
+    const { color, opacity, style, width } = this.state;
+    const colorScheme = ALL_COLOR_SCHEMES[this.props.colorScheme].slice();
+    if (color && color !== AUTOMATIC_COLOR &&
+      !colorScheme.find(x => x.toLowerCase() === color.toLowerCase())) {
+      colorScheme.push(color);
+    }
+    return (
+      <div>
+        <PopoverSection
+          isSelected
+          onSelect={() => {}}
+          title="Display configuration"
+          info={
+            'Configure your how you overlay is displayed here.'
+          }
+        >
+          <SelectControl
+            name="annotation-layer-stroke"
+            label="Style"
+            // see '../../../../visualizations/nvd3_vis.css'
+            options={[
+              { value: 'solid', label: 'Solid' },
+              { value: 'dashed', label: 'Dashed' },
+              { value: 'wideDashed', label: 'Wide Dashed' },
```

Comment:
I would call this 'long dashed', wide makes it sound like it affects the stroke width which it does not.